### PR TITLE
[44] Product Page using Html 

### PR DESCRIPTION
**Pull request by gracioust**
## Task: `Product Page using Html `
- `Removed pictures not been used`

### CODE REVIEW

File: submissions/tosinIshola/HTML/Ecommerce Website.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/tosinIshola/HTML/Ecommerce Website.html:1]: Well done @gracioust 

Is there a reason why you created this file also? If it is done by mistake, kindly remove it as part of your PR

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/tosinIshola/HTML/Ecommerce Website.html (hunk 1 above)
Comment (gracioust) [reply submissions/tosinIshola/HTML/Ecommerce Website.html:1]: i changed the name of the html file

Comment (gracioust) [reply submissions/tosinIshola/HTML/Ecommerce Website.html:1]: I changed it from Ecommerce Website.html to Ecommerce.html. I don't know why it is showing

Comment (gracioust) [reply submissions/tosinIshola/HTML/Ecommerce Website.html:1]: can i close this PR and create another one?

Comment (ToluAkin) [inline submissions/tosinIshola/HTML/Ecommerce Website.html:1]: @gracioust I think you should remove this file. You created another version already

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**